mne/viz/backends/renderer.py

```diff
@@ -0,0 +1,60 @@
+"""Core visualization operations."""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Joan Massich <mailsik@gmail.com>
+#          Guillaume Favelier <guillaume.favelier@gmail.com>
+#
+# License: Simplified BSD
+
+import os
+import importlib
+from ..utils import logger
+
+default_3d_backend = 'mayavi'
+
+
+try:
+    if MNE_3D_BACKEND is None:
+        MNE_3D_BACKEND = os.environ.get('MNE_3D_BACKEND',
+                                        default_3d_backend)
```

Comment:
Use `get_config('MNE_3D_BACKEND', default_3d_backend)` in these two calls -- it polls `environ` followed by the MNE config file.